src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -20,83 +20,133 @@
 
 helps['iot dps access-policy'] = """
 type: group
-short-summary: Manage Azure IoT Hub Device Provisioning Service access policies.
+short-summary: Manage access policies for an Azure IoT Hub Device Provisioning Service instance.
 """
 
 helps['iot dps access-policy create'] = """
 type: command
-short-summary: Create a new shared access policy in an Azure IoT Hub device provisioning service.
+short-summary: Create a new shared access policy in an Azure IoT Hub Device Provisioning Service instance.
 examples:
-  - name: Create a new shared access policy in an Azure IoT Hub device provisioning service with EnrollmentRead right
+  - name: Create a new shared access policy in an Azure IoT Hub Device Provisioning Service instance with EnrollmentRead right
     text: >
         az iot dps access-policy create --dps-name MyDps --resource-group MyResourceGroup --name MyPolicy --rights EnrollmentRead
 """
 
 helps['iot dps access-policy delete'] = """
 type: command
-short-summary: Delete a shared access policies in an Azure IoT Hub device provisioning service.
+short-summary: Delete a shared access policies in an Azure IoT Hub Device Provisioning Service instance.
 examples:
-  - name: Delete shared access policy 'MyPolicy' in an Azure IoT Hub device provisioning service
+  - name: Delete shared access policy 'MyPolicy' in an Azure IoT Hub Device Provisioning Service instance 'MyDps'
```

Comment:
super nit: I think   "....in **the** Azure IoT Hub Device Provisioning Service instance 'MyDPS'" is more grammatically correct since its a specific DPS but I don't feel strongly about it.

Reply:
It is more gramatically correct